### PR TITLE
SNOW 204166 ocsp telemetry update odbc

### PR DESCRIPTION
**Pull request by sfc-gh-hchaturvedi**
This PR aims at adding more specific OCSP related telemetry to better help in diagnostics.

### CODE REVIEW

**Review by sfc-gh-vreddy — COMMENTED** [review]
LGTM, bump up the version of oob in build_oob.sh and bat file

**Comment by sfc-gh-vreddy** [comment]
The tests are not being scheduled, something wrong with git work flow?

**Comment by sfc-gh-hchaturvedi** [comment]
I found some bugs that I am fixing in here. Will reach out once done. I created this PR to have some tests run as I am finding it hard to run them on my dev environment.

File: deps/oob-1.0.3/sf_ocsp_telemetry_data.c [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+void sf_otd_set_event_type(char *event_type, SF_OTD* ocsp_telemetry_data)
+{
+  if (!ocsp_telemetry_data)
+  {
+    return;
+  }
+
+  snprintf(ocsp_telemetry_data->event_type,OCSP_TELEMETRY_EVENT_MAX_LEN, event_type);
+}
+
```
Comment (sfc-gh-kwagner) [inline deps/oob-1.0.3/sf_ocsp_telemetry_data.c:25]: Why aren't all of the functions in this file inline functions?

**Review by sfc-gh-kwagner — APPROVED** [review]
LGTM other than inline comment

File: deps/oob-1.0.3/sf_ocsp_telemetry_data.c (hunk 1 above)
Comment (sfc-gh-hchaturvedi) [reply deps/oob-1.0.3/sf_ocsp_telemetry_data.c:25]: They could have been inline, I thought I'd just create a new placeholder if we want to add more functionality later. Wouldn't want to pollute the header file. It wouldn't harm much to put them inline but that's what the thought process was behind the decision.

**Pull request merged**